enterprise/backend/test/metabase_enterprise/advanced_permissions/api/group_manager_test.clj

```diff
@@ -28,26 +28,44 @@
               (update-group [user status group]
                 (testing (format ", update group with %s user" (mt/user-descriptor user))
                   (let [new-name (mt/random-name)]
-                    (mt/user-http-request user :put status (format "permissions/group/%d" (:id group)) {:name new-name}))))]
+                    (mt/user-http-request user :put status (format "permissions/group/%d" (:id group)) {:name new-name}))))
+
+              (delete-group [user status group-manager?]
+                (testing (format ", delete group with %s user" (mt/user-descriptor user))
+                  (let [user-id (u/the-id (if (keyword? user) (mt/fetch-user user) user))]
```

Comment:
When would `user` not be a keyword?

Reply:
lots, here is an example: https://github.com/metabase/metabase/blob/f730f6afba3713de486b75d0fea455c0b3a1e725/enterprise/backend/test/metabase_enterprise/advanced_permissions/api/group_manager_test.clj#L52